nbproject/project.properties

```diff
@@ -30,24 +30,23 @@ dist.jar=${dist.dir}/libai.jar
 dist.javadoc.dir=${dist.dir}/javadoc
 endorsed.classpath=
 excludes=
-file.reference.mysql-connector-5.0.5.jar=../../libs/mysql/mysql-connector-5.0.5.jar
 includes=**
 jar.compress=false
 javac.classpath=\
-    ${libs.swing-layout.classpath}:\
-    ${file.reference.mysql-connector-5.0.5.jar}
```

Comment:
This dependency is required for decision trees at the moment. In the decision tree package there is an interesting structure called DataSet that i want to use for the training of neural networks as it allows to do creation of training sets using cross references percentages.